src/renderer/components/distraction-settings/distraction-settings.js

```diff
@@ -45,6 +45,18 @@ export default Vue.extend({
     },
     hideActiveSubscriptions: function () {
       return this.$store.getters.getHideActiveSubscriptions
+    },
+    hideDescription: function () {
+      return this.$store.getters.getHideDescription
+    },
+    hideComments: function () {
+      return this.$store.getters.getHideComments
```

Comment:
```suggestion
    hideVideoComments: function () {
      return this.$store.getters.getHideVideoComments
```

Reply:
there's a property called hideCommentLikes and I feel like this change will make the code more inconsistent.